ivy_tests/test_ivy/test_frontends/test_torch/test_pooling_functions.py

```diff
@@ -66,31 +80,31 @@ def test_torch_avg_pool1d(
         min_side=1,
         max_side=4,
     ),
```

Comment:
Afair, the `arrays_for_pooling` now has two arguments `explicit_or_str_padding` and `only_explicit_padding`. Tweaking their values should give you the explicit padding output instead of a string padding output. If that works then you wouldn't have to generate `str` paddings and convert them to explicit ones yourself, kind of how it's implemented in `avg_pool3d`. Can you maybe try that out?

Reply:
@hmahmood24 I actually did try that out, but the problem is this generates padding values which are not necessarily 'same' or 'valid' paddings. In that case our torch frontend will default to applying a valid padding, but the native torch function will continue applying the padding as defined - so there will be a shape mismatch. I guess this is a flaw with our ivy functions expecting a string padding. But if we're fine with this we can just keep the testing as I've done it and it will only check padding values which actually are 'same' or 'valid'.

Reply:
@Sam-Armstrong Right that makes sense. Thanks for the clarification. Although, given that `paddle` supports `str` or `numerical` padding, while `tensorflow` only supports `str` padding and `torch` supports `numeraical` one, I think it's fair to add this functionality to `ivy` itself such that `ivy` supports padding in all of these formats since we strive to implement the superset behaviour of all the other frameworks. Feel free to create a Trello card for this and/or notify the Framework team on Discord of the matter so someone can work on this, if not yourself. Thanks Sam 😃